tailcfg/tailcfg.go

```diff
@@ -2171,6 +2172,16 @@ const (
 	// NodeAttrDNSForwarderDisableTCPRetries disables retrying truncated
 	// DNS queries over TCP if the response is truncated.
 	NodeAttrDNSForwarderDisableTCPRetries NodeCapability = "dns-forwarder-disable-tcp-retries"
+
+	// NodeAttrLinuxMustUseIPTables forces Linux clients to use iptables for
+	// netfilter management.
+	// This cannot be set simultaneously with NodeAttrLinuxMustUseNfTables.
+	NodeAttrLinuxMustUseIPTables NodeCapability = "linux-must-use-iptables"
```

Comment:
It might be worth considering the naming convention from `"one-cgnat?v=true"` above, i.e. `"netfilter?v=iptables"` and `"netfilter?v=nftables"`